tests/sentry/statistical_detectors/test_algorithm.py

```diff
@@ -186,6 +186,26 @@ def test_moving_average_detector_state_from_redis_dict_error(data, error):
         MovingAverageDetectorState.from_redis_dict(data)
 
 
+@pytest.mark.parametrize(
+    ["baseline", "rel_threshold", "auto_resolve"],
+    [
+        pytest.param(100, 0.1, True, id="equal"),
+        pytest.param(105, 0.1, True, id="within threshold above"),
+        pytest.param(95, 0.1, True, id="within threshold below"),
+        pytest.param(115, 0.1, False, id="exceed threshold above"),
```

Comment:
With this test, are we saying that if the EMA drops below the relative threshold factor, we won't mark it as auto-resolved? It's only resolved if the range that the new behaviour is exhibiting is within the baseline +/- the threshold around it?

I imagine that dropping well below would be a potential auto-resolution event

Reply:
That's a good point, if it ever dips below 1.1x time the baseline, I think that's a reason to resolve it. Let me fix that up